.github/workflows/cromwell-test-run.yml

```diff
@@ -22,11 +30,5 @@ jobs:
         run: wget -q https://github.com/broadinstitute/cromwell/releases/download/86/cromwell-86.jar
       -
         name: Execute Test Run of WDL Workflows
-        run: |
-          java -jar cromwell-86.jar run helloHostname/helloHostname.wdl
-          java -jar cromwell-86.jar run helloModuleHostname/helloModuleHostname.wdl
-          java -jar cromwell-86.jar run helloDockerHostname/helloDockerHostname.wdl
-
-
-
+        run: java -jar cromwell-86.jar run ${{ matrix.wdl }}
```

Comment:
i forget, when does `run`  exit? submits and then exits, or waits for a certain status before exioting?

Reply:
`run` exits once the workflow hits a final state, whether that's success or failure (can't remember the exact status wording of those two though...)

Reply:
ok thanks. just curious

Reply:
> submits and then exits, or waits for a certain status before exiting?

It waits for sure, if the command returns not `0` then it fails the whole action. You have to add a special arg to allow it to proceed in the event of a failure